cmd/skaffold/app/cmd/parse_config.go

```diff
@@ -108,11 +108,18 @@ func getConfigs(cfgOpts configOpts, opts config.SkaffoldOptions, r *record) ([]*
 func processEachConfig(config *latest.SkaffoldConfig, cfgOpts configOpts, opts config.SkaffoldOptions, r *record, index int) ([]*latest.SkaffoldConfig, error) {
```

Comment:
Would be helpful to describe what `index` represents here.  (And `it's` &rarr; `its`)

Reply:
fixed typo; added description